### DISCUSSION TITLE
조직 이름 변경내역 및 조직 코드 변경내역도 만들어야 할까요?

### DISCUSSION THREAD

**Discussion by haedanui** [nhnacademy-aiot1-5/api-service#43] [Ideas]
제곧내

**Comment by caboooom** [comment]
조직이름은 변경내역 남겨도 좋을 것 같구
조직코드는 일회성이라서 굳이 남겨야 할까 싶습니다

**Comment by huhsiyoung** [comment]
무조건 있어야하는 기능은 아닌 거 같아요
있으면 관리할 때 편하고 까먹었을 때 알기 좋습니다

**Comment by codethestudent** [comment]
데이터베이스에 이미 있는 데이터에 그냥 추가하고 추가된 새이름을 쓰면 기록도 님고 해결되지 않나 라고 생각했는데 만약 이름이나 코드를 unique 설정을 걸어두면 중복 때문에 테이블을 새로 만들고  해야하는 복잡함이 있을것 같네요.. 그럴바엔 굳이 남길 필요 없을것 같습니다
